react_on_rails.gemspec

```diff
@@ -27,4 +27,5 @@ Gem::Specification.new do |s|
   s.add_development_dependency "rake", "~> 10.0"
   s.add_development_dependency "rspec"
   s.add_development_dependency "coveralls"
+  s.add_development_dependency "generator_spec"
```

Comment:
ATTENTION: added development gem dependency. This wraps Rails::Generators::Testing helpers so we can use it in RSpec